server/src/database.js

```diff
@@ -0,0 +1,29 @@
+import SQLiteDatabase from './database/database-sqlite'
+import { config } from './globals'
+
+let db = null
+
+export async function initializeDatabase() {
+  db = new SQLiteDatabase(config.sqliteFilename)
+  await db.initialize(config.dev)
+}
+
+export async function getComments() { return db.query('SELECT * FROM comments') }
+
+export async function getQuestions() { return db.query('SELECT * FROM questions') }
+
+export async function getReactions() { return db.query('SELECT * FROM reactions') }
+
+export async function addReaction(values = []) { return db.run('INSERT INTO reactions(id, emoji, user, comment_id) VALUES (?, ?, ?, ?)', values) }
```

Comment:
Should these be like `addReaction(...values) { }`
would make calling code look like `addReaction(a, b, c)` instead of `addReaction([a, b, c])`.

Also maybe worth considering for more safety:
```js
function addReaction({ id, emoji, user }) { return db.run(..., [id, emoji, user]) }

addReaction({ id: 3, emoji: ':ok_hand', user: '1234' })
```